go/go.mod

```diff
@@ -1,3 +1,11 @@
 module interview
 
 go 1.21.2
+
+require github.com/stretchr/testify v1.8.4
```

Comment:
MIT license, since this is public we probably should include the license file somewhere

Reply:
Otherwise looks good to me

Reply:
That's a great callout - I'm not 100% sure what the right move is on this one. I was under the impression that if a dependency we utilize is merely imported and we don't redistribute it as a binary that we're ok to not include the license. 

If I go peek at Kubernetes, they have a whole suite of licenses and list this testing one there. https://github.com/kubernetes/kubernetes/tree/master/LICENSES/vendor/github.com/stretchr/testify

If I search around GitHub for their copyright I do see it in several places. I'll probably look at adding some sort of 3rd party dependencies section and try and get the license added in there.